mmv1/templates/terraform/constants/dlp_stored_info_type.go.erb

```diff
@@ -0,0 +1,36 @@
+<%# The license inside this block applies to this file.
+	# Copyright 2023 Google Inc.
+	# Licensed under the Apache License, Version 2.0 (the "License");
+	# you may not use this file except in compliance with the License.
+	# You may obtain a copy of the License at
+	#
+	#     http://www.apache.org/licenses/LICENSE-2.0
+	#
+	# Unless required by applicable law or agreed to in writing, software
+	# distributed under the License is distributed on an "AS IS" BASIS,
+	# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	# See the License for the specific language governing permissions and
+	# limitations under the License.
+-%>
+func storedInfoTypeCustomizeDiffFunc(diff TerraformResourceDiff) error {
```

Comment:
Could we add a comment here to explain why this is needed? It could mention that updates can be done on these fields if they are changed, but not if the user changes between dictionary/regex/large_custom_dictionary.

Reply:
I've added comments for this customizedDiff in this [PR](https://github.com/GoogleCloudPlatform/magic-modules/pull/7634).